clients/cpp/public/myanmartools.h

```diff
@@ -35,4 +40,24 @@ class ZawgyiDetector {
 
 }  // namespace google_myanmar_tools
 
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+// C bindings (the "GMT" prefix stands for "Google Myanmar Tools")
+typedef void *GMTZawgyiDetector;
```

Comment:
Suggestion: if you use

```c
typedef struct GMTZawgyiDetector GMTZawgyiDetector;
```

then your C code uses the type `GMTZawgyiDetector*`, and you get more type safety.  It also makes it more clear that this is a pointer that needs to be cleaned up.

https://stackoverflow.com/a/3965308/1407170

Reply:
Done, thanks for the tip.